problems/test/problem1_test.rb

```diff
@@ -0,0 +1,26 @@
+require "minitest/autorun"
+require_relative "../lib/problem1"
+
+class Problem1Test < MiniTest::Unit::TestCase
+  def setup
+    @problem1 = Problem1.new
+  end
+
+  def test_given_empty_array_sum_is_zero

```

Comment:
I like the name of all the tests.
Have you tried using Minitest's expectations syntax? 
They'll become even nicer as it's closer to normal speech